cob_gazebo_worlds/urdf/empty/empty.urdf.xacro

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro"
+       name="empty">
+
+  <link name="world"/>
```

Comment:
the floor comes with the ```empty.world``` passed to the ```gazebo_ros``` launch file [here](https://github.com/ipa-fxm/cob_simulation/blob/89efae492b1461ecd071c9df9026fc27e250a40d/cob_gazebo_worlds/launch/world.launch#L27)